activities/activities.go

```diff
@@ -0,0 +1,8 @@
+package activities
```

Comment:
For a standard project I understand why you say that, in this case given that it's specifically demonstrating Temporal I think it's better to call them activities to make it clear how the code is structured relative to Temporal primitives.

Reply:
But what I'm saying is it may be more reasonable to structure relative to an idiomatic Go project using Temporal rather than around Temporal primitives.